packages/framework/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { Client } from './client';
 export * from './constants';
+export * from './errors';
```

Comment:
We want to expose all errors to the public API because they can be used to build exhaustive error handling checks.